### PR TITLE
feat(posts): Golab 2024 IT

### PR DESCRIPTION
**Pull request by t-geindre**
- [x] Add thumbnail
- [x] Add description
- [x] Complete author section
- [x] Add photos
- [x] Add links

### CODE REVIEW

File: _posts/2024-11-12-golab-florence.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+layout: post
+title: Insights from Golab Conference 2024, Florence
+description: TODO
+author: [t_geindre, k_phan]
+tags: [golang, opensource, community, conference, 2024]
+color: rgb(251,87,66)
+thumbnail: "/images/posts/2024-11-12-golab-firenze/firenze.jpg"
+---
+
+Attending the Golab conference in Florence has been a thrilling experience for us as developers. Immersing ourselves in talks by some of the most brilliant minds in the Go community gave us a deeper understanding of advanced concepts, practical libraries, and the future of Go. We’ve summarized the key takeaways from the talks we attended to share some of the valuable lessons and fresh perspectives we gained.
```
Comment (mikaelrandy) [inline _posts/2024-11-12-golab-florence.md:11]:
```suggestion
Attending the [Golab conference](https://golab.io/) in Florence has been a thrilling experience for us as developers. Immersing ourselves in talks by some of the most brilliant minds in the Go community gave us a deeper understanding of advanced concepts, practical libraries, and the future of Go. We’ve summarized the key takeaways from the talks we attended to share some of the valuable lessons and fresh perspectives we gained.
```

File: _posts/2024-11-12-golab-florence.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+---
+
+## Keynote: Go Telemetry Wins
+
+The opening keynote, delivered by a member of the Go team, focused on the importance of telemetry in the Go ecosystem itself. The speaker explained how telemetry data helps the Go team make informed decisions to improve the language and its ecosystem, emphasizing the strict measures taken to ensure user privacy. We learned how telemetry is carefully designed to protect sensitive information while still providing valuable insights. The session concluded with a call to action, encouraging everyone to enable telemetry on their Go installations to support ongoing enhancements and community-driven improvements.
```
Comment (mikaelrandy) [inline _posts/2024-11-12-golab-florence.md:17]:
```suggestion
The opening keynote, delivered by Russ Cox,  a member of the Go team, focused on the importance of telemetry in the Go ecosystem itself. Russ explained how telemetry data helps the Go team make informed decisions to improve the language and its ecosystem, emphasizing the strict measures taken to ensure user privacy. We learned how telemetry is carefully designed to protect sensitive information while still providing valuable insights. The session concluded with a call to action, encouraging everyone to enable telemetry on their Go installations to support ongoing enhancements and community-driven improvements.
```

File: _posts/2024-11-12-golab-florence.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+---
+
+## Watermill: The Missing Standard Library for Event-Driven Applications
+
+The Watermill library was highlighted as a game-changer for event-driven architecture in Go. Initially born out of a need to simplify the complexity of building event-driven systems, Watermill has evolved into a mature and widely adopted library. The talk inspired us by demonstrating how adopting Watermill can transform projects needing scalability and resilience, and it has now become a key consideration for us in future architectural designs.
```
Comment (mikaelrandy) [inline _posts/2024-11-12-golab-florence.md:23]:
```suggestion
The [Watermill library](https://watermill.io/) was highlighted as a game-changer for event-driven architecture in Go. Initially born out of a need to simplify the complexity of building event-driven systems, Watermill has evolved into a mature and widely adopted library. The talk inspired us by demonstrating how adopting Watermill can transform projects needing scalability and resilience, and it has now become a key consideration for us in future architectural designs.
```

**Review by mikaelrandy — COMMENTED** [review]
Please add some links ;)

**Comment by t-geindre** [comment]
> Please add some links ;)
That was indeed in the todolist :wink:

File: _posts/2024-11-12-golab-florence.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+- Open source so it's tool agnostic
+![Opentelemetry simple schema](/images/posts/2024-11-12-golab-firenze/opentelemtry_schema.avif)
+
+At Bedrock, we mainly use [NewRelic for our monitoring](https://tech.bedrockstreaming.com/2024/10/03/enhancing-production-monitoring-with-newrelic.html), but we are free either to use the NewRelic agent or to use OpenTelemetry to collect traces, metrics and logs.
+
+---
+
+## Watermill: The Missing Standard Library for Event-Driven Applications
+
+The Watermill library was highlighted as a game-changer for event-driven architecture in Go. Initially born out of a need to simplify the complexity of building event-driven systems, Watermill has evolved into a mature and widely adopted library. The talk inspired us by demonstrating how adopting Watermill can transform projects needing scalability and resilience, and it has now become a key consideration for us in future architectural designs.
```
Comment (titoko) [inline _posts/2024-11-12-golab-florence.md:44]: **suggestion:** 
```suggestion
The Watermill library was highlighted as a game-changer for event-driven architecture in Go. Initially born out of a need to simplify the complexity of building event-driven systems, Watermill has evolved into a mature and widely adopted library. The talk inspired us by demonstrating how adopting Watermill can transform projects needing scalability and resilience, and it has now become a key consideration for us in future architectural designs.
The design of this very complete library is thought out in three blocks that can be used independently or not:
![pyramid](https://github.com/user-attachments/assets/ce78b1ab-6259-4fe1-91ec-fb421c3fdfdf)

The first, on which the rest is based, is a simple [Publisher/Subscriber] brick (https://watermill.io/docs/pub-sub/) with several implementations ( go channel,kafka,RabbitMQ etc...).
The second is a [Router] layer (https://watermill.io/docs/messages-router/) which manages advanced event dispatch processing and adds the possibility of using middleware such as metrics, poison queue, retrying, throttling, etc. .

The last block is the [CQRS] management block (https://watermill.io/docs/cqrs/).
The project documentation is well supplied and full of examples that make bootstrapping in existing projects much easier.

```

File: _posts/2024-11-12-golab-florence.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+## Keynote: Go Telemetry Wins
+
+The opening keynote, delivered by Russ Cox, focused on the importance of telemetry in the Go ecosystem itself. Russ explained how telemetry data helps the Go team make informed decisions to improve the language and its ecosystem, emphasizing the strict measures taken to ensure user privacy. We learned how telemetry is carefully designed to protect sensitive information while still providing valuable insights.
+
+![Go telemetry wins](/images/posts/2024-11-12-golab-firenze/go_telemetry_keynote.jpg)
+
+The session concluded with a call to action, encouraging everyone to enable telemetry (by running `go telemetry on`) on their Go installations to support ongoing enhancements and community-driven improvements.
+
+
```
Comment (titoko) [inline _posts/2024-11-12-golab-florence.md:23]: **suggestion:** 
tips: 
All data collected are public and could be found [here](https://telemetry.go.dev/)

**Review by kphan-epitech — APPROVED** [review]

**Pull request merged**